rslint_core/src/groups/errors/valid_typeof.rs

```diff
@@ -0,0 +1,120 @@
+use crate::rule_prelude::*;
+use ast::{BinExpr, Expr, Literal, UnaryExpr};
+
+declare_lint! {
+    /**
+    Enforce the use of valid string literals in a `typeof` comparison.
+
```

Comment:
I think a short sentence saying why an invalid value is bad would greatly benefit users, it may seem clear to us but not to beginners 👍